app/views/posts/index.html.erb

```diff
@@ -24,6 +24,8 @@
           </div>
         <% end %>
       </div>
+
+      <%= render 'likes/new', user: @user, post: post %>
     </div>
```

Comment:
- [ ] Please kindly take note that I got the error message screenshot below due to this line of code highlited 👇. 
 
<img width="673" alt="Untitled" src="https://github.com/BHObserver/blog-app/assets/103386183/a963c3a7-8db7-4050-9b1a-033ce1a48dcf">

- [ ] When I removed the line of code (28), the page was displayed but not properly implemented according to the requirement, kindly ensure that all the post are rendered in the `post_index` page and five most recents comments are expected to be rendered as well. 👍 

<img width="601" alt="Untitled" src="https://github.com/BHObserver/blog-app/assets/103386183/345a7268-6926-48da-8046-1589d3a307f0">